src/huggingface_hub/repocard_types.py

```diff
@@ -0,0 +1,83 @@
+# Code generated by jtd-codegen for Python v0.3.1
```

Comment:
Nice !
This could be useful to add in comments how to update it to propose a PR if/when the original schema changes ?